source/dotnet/wholesale-api/CalculationResults/CalculationResults.Interfaces/CalculationResults/Model/EnergyResults/EnergyResultQueryParameters.cs

```diff
@@ -17,11 +17,22 @@
 
 namespace Energinet.DataHub.Wholesale.CalculationResults.Interfaces.CalculationResults.Model.EnergyResults;
 
-public record EnergyResultQuery(
+public record EnergyResultQueryParameters(
```

Comment:
Is this being used for both energy results and aggregated time series? I anticipate no and thus recommend:
```suggestion
public record AggregatedTimeSeriesQueryParameters(
```